packages/form/typings/Checkbox.d.ts

```diff
@@ -1,12 +1,14 @@
 import * as React from 'react';
-import { InputProps } from './Input';
 
-interface CheckboxProps extends InputProps {
+interface CheckboxProps extends React.HTMLAttributes<HTMLInputElement> {
```

Comment:
I feel like we should still extend `InputProps` here.  Was it causing a problem?

Reply:
It said it was incorrectly extended because 'value' was incompatible. It looks like the same change was made to RadioProps a long while back to so I made the same change here.

Reply:
Ah ok. Let's just run with it then